pyppeteer/page.py

```diff
@@ -201,21 +184,89 @@ def _onTargetDetached(event: Dict) -> None:
         client.on('Runtime.exceptionThrown',
                   lambda exception: self._handleException(
                       exception.get('exceptionDetails')))
-        client.on('Security.certificateError',
-                  lambda event: self._onCertificateError(event))
         client.on('Inspector.targetCrashed',
                   lambda event: self._onTargetCrashed())
         client.on('Performance.metrics',
                   lambda event: self._emitMetrics(event))
         client.on('Log.entryAdded',
                   lambda event: self._onLogEntryAdded(event))
+        client.on('Page.fileChooserOpened',
+                  lambda event: self._onFileChooser(event))
 
         def closed(fut: asyncio.futures.Future) -> None:
-            self.emit(Page.Events.Close)
+            self.emit(Events.Page.Close)
             self._closed = True
 
         self._target._isClosedPromise.add_done_callback(closed)
 
+    async def _initialize(self):
+        await asyncio.gather(
+            self._frameManager.initiliaze(),
+            self._client.send('Target.setAutoAttach', {
+                'autoAttach': True,
+                'waitForDebuggerOnStart': False,
+                'flatten': True,
+            }
+                              ),
+            self._client.send('Performance.enable'),
+            self._client.send('Log.enable'),
+        )
+
+    async def _onFileChooser(self, event: dict):
```

Comment:
consistency in typings
```suggestion
    async def _onFileChooser(self, event: Dict):
```